accounting/transferwise/models.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2017-2018 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+TransferWise models to be used by the Accounting service.
+"""
+
+from django.core.exceptions import ValidationError
+from django.db import IntegrityError, models
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+
+from accounting.account.models import Account
+from accounting.common.models import CommonModel, UuidModel
+from accounting.invoice.models import Invoice
+from accounting.third_party_api.google.mixins import GoogleDriveMixin
+from accounting.transferwise.mixins import TransferWiseCsvMixin
+
+
+class TransferWiseBulkPayment(CommonModel, UuidModel, TransferWiseCsvMixin, GoogleDriveMixin):
+    """
+    A model that represents TransferWise bulk payments.
+
+    Can be used to generate bulk payment CSVs.
+    """
+
+    date = models.DateTimeField(
+        default=timezone.now,
+        help_text=_("The date this bulk payment was created and uploaded to TransferWise. "
+                    "Defaults to right now, but can be changed."))
+    sender = models.ForeignKey(
+        Account, on_delete=models.CASCADE, related_name='transferwise_sender_bulk_payments',
+        help_text=_("The sender of the TransferWise bulk payment. "
+                    "Refers to the account owner of the account initiating the payment."))
+    csv_path = models.URLField(
+        blank=True, null=True, max_length=300,
+        help_text=_("The absolute URL that can be used to retrieve the bulk payment CSV."))
+
+    # TODO: The following 3 fields are temporary.
+    # TODO: These will be automated through jobs at some point, generated at particular times of the month.
+    auto_create_payments_on_save = models.BooleanField(
+        default=False,
```

Comment:
I'm not sure if I already asked about this in the previous PR, but why are the defaults to those options False, when we will always need them to be True in our use cases? That generates a bunch of clicks every time, so is there a good reason for this?

Reply:
@antoviaque I wrote that in from a dev perspective: it's easier to test it when you have a blank slate. But yeah, I get your point, from an admin perspective.

No worries, #10 is erasing these things. It won't be erasing the TransferWise ones just yet actually, but if I have time I will try to fit that in, and if not the new ticket, as mentioned above, is 1-2 points worth of work hopefully.

In the meantime, since Alicia will just be creating *1* of these, it's not too bad for 3-5 seconds, and will prevent me having to change up a bunch of test code for an hour.

Reply:
@UmanShahzad Yup, in that case it's fine 👍